src/main/java/org/cloudfoundry/promregator/lite/discovery/DiscoveryControllerV2.kt

```diff
@@ -48,8 +50,36 @@ class DiscoveryControllerV2(
                   @RequestParam application: String?
     ): List<DiscoveryV2Response> {
```

Comment:
I'll add a MockMVC test for this endpoint shortly. But didn't have it ready yet.